src/components/utils/include/utils/optional.h

```diff
@@ -79,5 +88,86 @@ class Optional {
   bool is_initialized_;
 };
 
+template <typename ObjectType, typename ErrorType>
+class Optional<ObjectType, ErrorType, StoragePolicy_CopyValue> {
```

Comment:
Looks redundant, given the existing `Optional` class. As mentioned, you can use the `boost::optional` class.

Reply:
@jacobkeeler Thanks. Fixed in b07545f011f59e3f00851e5a6db7980a7cd76258